src/components/deploymentConfig/ReadmeConfig.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { useEffect, useState } from 'react';
+import { useKeyDown } from '../common';
+import CodeEditor from '../CodeEditor/CodeEditor';
+import checkGreen from '../../assets/icons/misc/checkGreen.svg';
+import arrowSquareout from '../../assets/icons/misc/ArrowSquareOut.svg';
+import { MarkDown } from '../charts/discoverChartDetail/DiscoverChartDetails';
+import './deploymentConfig.scss';
+
+interface readme {
+    readme: any;
+    value: string;
+    handleClose: any;
+    loading: boolean;
+    height?: number;
+    onChange: any;
+    schema?: any;
+    readOnly?: boolean;
+    defaultValue?: string;
+    mode?: "yaml" | "json" | "shell";
+}
+
+function ReadmeConfig({ readme, value, handleClose, loading, height, onChange, schema, readOnly, defaultValue, mode }:readme) {
+    const key = useKeyDown();
+    const [resp, setTempForm] = useState();
+    useEffect(() => {
+        if (key.join().includes('Escape')) {
+            handleClose();
+        }
+    }, [key.join()]);
+
+    const handle = () => {
```

Comment:
Name convention should be more specified, Unable to get what are you handling here